src/renderer/views/Watch/Watch.js

```diff
@@ -395,8 +396,38 @@ export default Vue.extend({
             if (typeof startTimestamp !== 'undefined') {
               const upcomingTimestamp = new Date(result.videoDetails.liveBroadcastDetails.startTimestamp)
               this.upcomingTimestamp = upcomingTimestamp.toLocaleString()
+
+              let upcomingTimeLeft = upcomingTimestamp - new Date()
+              let timeUnit
+              if (upcomingTimeLeft <= 0) {
+                this.upcomingTimeLeft = this.$t('Video.Starting Soon')
+              } else {
+                upcomingTimeLeft = upcomingTimeLeft / 1000
+                timeUnit = 'Second'
+                if (upcomingTimeLeft > 60) {
+                  upcomingTimeLeft = upcomingTimeLeft / 60
+                  timeUnit = 'Minute'
+                  if (upcomingTimeLeft > 120) {
```

Comment:
Haven't tested yey, but shouldn't this be 60 instead of 120? (Correct me if I'm wrong)

Reply:
AFAIK youtube switches to showing time left in minutes at 120 minutes remaining. I tried to replicate how they choose to display it.